client/ayon_substancepainter/api/lib.py

```diff
@@ -272,6 +272,10 @@ def _filename_no_ext(path):
         "$udim": "([0-9]{4})"
     }
 
+    version_info = substance_painter.application.version_info()
+    if version_info >= (11, 0, 1):
+        key_matches["$uvTileName"] = r"(\s*[^\W_](?:[\w -]*(?<! )))"
```

Comment:
Could you elaborate on this regex and add a comment about this. What is this matching?

Reply:
Added the comments to explain the pattern and its limitation.

Reply:
> Can we query the UV tilenames that are in the project or alike? If so, can't we match it much more explicitly? This feels very unsafe/arbitrary currently.

I have asked the Adobe about that and see there is any suggestion for the `$uvTileName`. They still do not reply me yet. And the latest docs of Python API does not show any relevant function which query the uv tilenames. 
We can mention about this as a limtiation.

Reply:
As discussed in private chat - you can query the tilenames from Substance Painter's API - so it's best to use the explicit values from there.